controller/src/beerocks/master/tasks/dynamic_channel_selection_task.cpp

```diff
@@ -252,6 +252,15 @@ void dynamic_channel_selection_task::handle_event(int event_type, void *obj)
     }
     case eEvent::SCAN_TRIGGER_FAILED: {
         TASK_LOG(DEBUG) << "SCAN_TRIGGER_FAILED received";
+        if (fsm_in_state(eState::WAIT_FOR_SCAN_TRIGGERED)) {
+            auto scan_trigger_failed_event = reinterpret_cast<sScanEvent *>(obj);
+            event_handled                  = true;
+            TASK_LOG(DEBUG) << "SCAN_TRIGGER_FAILED handled on:"
+                            << scan_trigger_failed_event->radio_mac.oct;
```

Comment:
That's not a very informative message, and in any case I think it should be printed at least with an INFO (if not WARNING) level.
Consider changing the message to something like:
"Triggering scan failed on <MAC>, aborting scan."